### PR TITLE
net-analyzer/netdata-1.3.0: version bump

### PR DESCRIPTION
**Pull request by candrews**
https://bugs.gentoo.org/show_bug.cgi?id=592352

### CODE REVIEW

**Comment by candrews** [comment]
Upstream now requires python for a lot of functionality so the ebuild now conditionally depends on it. I think I did the dependencies correctly (upstream can use any version of Python 2 or 3).

File: net-analyzer/netdata/netdata-1.3.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	default
+	[[ ${PV} == "9999" ]] && eautoreconf
+}
+
+src_configure() {
+	econf \
+		--localstatedir=/var \
+		--with-user=${NETDATA_USER} \
+		$(use_enable nfacct plugin-nfacct) \
+		$(use_enable cpu_flags_x86_sse sse)
```
Comment (SoapGentoo) [inline net-analyzer/netdata/netdata-1.3.0.ebuild]: missing a trailing backslash, without which the `compression` use flag never gets honoured

File: net-analyzer/netdata/netdata-1.3.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+		$(use_with compression zlib)
+}
+
+src_install() {
+	default
+
+	fowners ${NETDATA_USER}:${NETDATA_GROUP} /var/log/netdata
+	fowners ${NETDATA_USER}:${NETDATA_GROUP} /var/cache/netdata
+
+	chown -Rc ${NETDATA_USER}:${NETDATA_GROUP} "${ED}"/usr/share/${PN} || die
```
Comment (SoapGentoo) [inline net-analyzer/netdata/netdata-1.3.0.ebuild]: don't use `chown`, use Gentoo's internal `fowners` instead. You can then remove the `|| die` command and the `"${ED}"/` prefix.

In related news, also remember: `${ED}` is slash-terminated, meaning that you should normally use it without an adjacent slash.

**Comment by gktrk** [comment]
@gentoo/proxy-maint

File: net-analyzer/netdata/netdata-1.3.0.ebuild (hunk 2 above)
Comment (candrews) [inline net-analyzer/netdata/netdata-1.3.0.ebuild]: TIL Thank you!

File: net-analyzer/netdata/netdata-1.3.0.ebuild (hunk 1 above)
Comment (candrews) [inline net-analyzer/netdata/netdata-1.3.0.ebuild]: Thanks for pointing that out!

**Pull request closed without merging**